resources/base-url-target.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```

Comment:
This should be in `tests/wpt/web-platform-tests/html/semantics/forms/the-form-element/resources/`, not servo's toplevel resources directory :)